components/usage/pkg/apiv1/billing.go

```diff
@@ -118,7 +108,8 @@ func (s *BillingService) FinalizeInvoice(ctx context.Context, in *v1.FinalizeInv
 		ID:            uuid.New(),
 		AttributionID: attributionID,
 		Description:   fmt.Sprintf("Invoice %s finalized in Stripe", invoice.ID),
-		CreditCents:   db.NewCreditCents(float64(creditsOnInvoice)),
+		// Apply negative value of credits to reduce accrued credit usage
+		CreditCents:   db.NewCreditCents(float64(-creditsOnInvoice)),
```

Comment:
Ideally we'd do it the otherway around. Maybe we can create a PR with a migration that negates all credits and changes the respective code as well?

Reply:
Actually thinking about it again, it might be fine as is. After all the table is called usage so positive means used negative means gained. 🤷

Reply:
We can still make the change and drop the usage records manually

Reply:
Spoke directly, we'll be keeping it as is to more closely align with Stripe which also doesn't want negative credits, but positive credits for usage